src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepositoryImpl.java

```diff
@@ -93,8 +93,18 @@ public OverallStatisticsContent overallStatisticsContent(Filter filter, Sort sor
 				.fetch());
 	}
 
+	private List<JTestItemTypeEnum> getItemTypeCriteria(boolean includeMethods) {
```

Comment:
Can we make it returning Condition, provide some docs and use it directly in jooq builder?